frontends/web/src/components/view/view.tsx

```diff
@@ -41,6 +42,7 @@ type TViewProps = {
  * @param minHeight optional minimum height, useful for keeping content area same size through multiple views
  * @param onClose if a callback is provided it will render a close button that triggers the callback
  * @param textCenter centers all text content in the view
+ * @param verticallyCentered centers all text content in the view
```

Comment:
verticallyCentered make sense

the docstring text implies it should also work also for fullscreen, which would actually be nice.

I made a POC with a few other changes: https://github.com/shonsirsha/bitbox-wallet-app/pull/1  wdyt?

Reply:
> the docstring text implies it should also work also for fullscreen, which would actually be nice.

I don't think I can see any difference when `fullScreen` and `verticallyCentered` are both `true` as opposed just having `fullScreen` to be true (setting fullScreen automatically vertically centres it..?)